### ISSUE TITLE
imgui doesn't render on linux.

### ISSUE THREAD

**Issue by JodiTheTigger** [irlanrobson/bounce#3]
tl;dr: Don't use glMatrix* functions, change gl version to minimum of version 3.30 (https://en.wikipedia.org/wiki/OpenGL_Shading_Language)

------

Imgui wasn't rendering, this was because my default context is opengl 3.0, which didn't support shader model version 3.3 used in `imgui_impl_glfw_gl3.cpp:211`

So I hacked the code to use 4.3
```
    glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 4);
    glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
```
And now the gui renders, but the objects don't.

This is because you are using the _removed_ `glMatrix*` functions. Which fails under the 3.3 and above context.
I don't have the compatibility context, so I cannot use both imgui and your rendering code at the same time.

**Comment by irlanrobson** [comment]
Require OpenGL 3.0.
From imgui_impl_glfw_gl3.cpp:211 edit #version 330 to #version 130.
Recompile.

Does this solves the problems for you?

These solved these problems for me on Ubuntu x64 16.04, Mesa 11.2.0, gmake, debug_x64.

**Comment by JodiTheTigger** [comment]
Yup, that fixes it :-)
